modules/luci-mod-network/htdocs/luci-static/resources/view/network/dhcp.js

```diff
@@ -615,20 +619,33 @@ return view.extend({
 			uci.unset('dhcp', section, 'dns');
 		};
 
-		so = ss.option(form.Value, 'mac', _('MAC address'));
-		so.datatype = 'list(macaddr)';
+		so = ss.option(form.Value, 'mac',
+			_('MAC address(es)'),
+			_('The hardware address(es) of this entry/host, separated by spaces.'));
+		so.validate = function(section_id, value) {
+			var macaddrs = L.toArray(value);
+
+			for (var i = 0; i < macaddrs.length; i++)
+				if (!macaddrs[i].match(/^([a-fA-F0-9]{2}|\*):([a-fA-F0-9]{2}:|\*:){4}(?:[a-fA-F0-9]{2}|\*)$/))
+				return _('Expecting a valid MAC address, optionally including wildcards');
+
+			return true;
+		};
 		so.rmempty  = true;
 		so.cfgvalue = function(section) {
 			var macs = L.toArray(uci.get('dhcp', section, 'mac')),
-			    result = [];
+				result = [];
```

Comment:
Incorrect indentation @jow- prefers spaces for the variable declaration.
See https://github.com/openwrt/luci/blob/0640e487b5db524a29dff4b325bd458de0f0be7b/modules/luci-mod-network/htdocs/luci-static/resources/view/network/dhcp.js#L653